components/list/bullet/src/index.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import cx from 'classnames'
+
+const ListBullet = ({listItems, linkFactory, responsive}) => {
+  const listBulletClass = cx('sui-ListBullet', {
+    'sui-ListBullet--responsive': responsive
+  })
+
+  return (
+    <div className={listBulletClass}>
+      {listItems &&
+        listItems.map((item, index) => (
+          <div key={index} className="sui-ListBullet-item">
+            <img
+              className="sui-ListBullet-illustration"
+              src={item.illustration}
+            />
+            <div className="sui-ListBullet-description">
+              {item.title && (
```

Comment:
Related to the question below, making the shape `isRequired`, does it make all the `keys` required?
If so, we do not need this check

In negative case, we should think the make `illustration` and `title` required IMHO

Reply:
fixed!
> requiered are`illustration` and `description`.